gap_follow:
  ros__parameters:
    should_plot: False
    max_speed: 2.0
    car_width: 0.8
    fov_angle: 70
